sentry/src/main/java/io/sentry/SentryClient.java

```diff
@@ -400,6 +400,15 @@ public void captureSession(final @NotNull Session session, final @Nullable Objec
     if (transaction.getEnvironment() == null) {
       transaction.setEnvironment(options.getEnvironment());
     }
+    if (transaction.getTags() == null) {
+      transaction.setTags(new HashMap<>(options.getTags()));
+    } else {
+      for (Map.Entry<String, String> item : options.getTags().entrySet()) {
+        if (!transaction.getTags().containsKey(item.getKey())) {
+          transaction.setTag(item.getKey(), item.getValue());
+        }
+      }
+    }
```

Comment:
what's about the tags set on the scope? should we also copy them?

cc @bruno-garcia

Reply:
oof, yeah we're missing that. All tags should be added